src/validators/tasks.py

```diff
@@ -283,3 +365,18 @@ def _get_validators_amounts(vault_assets: int, validator_type: ValidatorType) ->
     if vault_assets >= MIN_ACTIVATION_BALANCE:
         amounts.append(Gwei(int(Web3.from_wei(vault_assets, 'gwei'))))
     return amounts
+
+
+def _get_funding_amounts(
+    vault_validators: list[ConsensusValidator], amount: Gwei
+) -> dict[HexStr, Gwei]:
+    vault_validators.sort(key=lambda x: x.balance, reverse=True)
+    result = {}
+    for validator in vault_validators:
+        if MAX_EFFECTIVE_BALANCE_GWEI - validator.balance > 0:
+            val_amount = min(MAX_EFFECTIVE_BALANCE_GWEI - validator.balance, amount)
+            result[validator.public_key] = Gwei(val_amount)
```

Comment:
I think you should also check here whether `val_amount >= MIN_DEPOSIT_AMOUNT_GWEI`